app-misc/ddcutil/ddcutil-0.8.4.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools linux-info udev user
+
+DESCRIPTION="Program for querying and changing monitor settings"
+HOMEPAGE="http://www.ddcutil.com/"
+
+SRC_URI="https://github.com/rockowitz/ddcutil/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+# Binary drivers need special instructions compared to the open source counterparts.
+# If a user switches drivers, they will need to set different use flags for
+# Xorg or Wayland or Mesa, so this will trigger the rebuild against
+# the different drivers.
+# Remove ATI/AMD driver since it's masked for removal.
+# Will most likely need to set this for AMDGPU when in portage.
+IUSE="usb-monitor user-permissions video_cards_nvidia"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="x11-libs/libXrandr
+	 x11-libs/libX11
+	 dev-libs/glib:2
+	 sys-apps/i2c-tools
+	 virtual/udev
+	 usb-monitor? (
+		dev-libs/hidapi
+		virtual/libusb:1
+		sys-apps/usbutils )"
+DEPEND="virtual/pkgconfig
+	${RDEPEND}"
+
+pkg_pretend() {
+	# This program needs /dev/ic2-* devices to communicate with the monitor.
+	CONFIG_CHECK="I2C_CHARDEV"
```

Comment:
Make those non-fatal. We only use fatal when we're building kernel modules that will fail to build otherwise. For runtime dep, we make them non-fatal as people can build stuff on a build server, with a kernel they're not going to use, or they can just update the kernel after installing the package.